guide-us-english.tex

```diff
@@ -199,73 +199,51 @@ \subsection{Team-Level Process}
 \item operate in a way that is sustainable and enriching for the team.
 \end{itemize}
 
 \subsection{Coordinating the ``How'' - The Scrum of Scrums}
-A set of the teams that have a need to coordinate comprise a
-\textbf{``Scrum of Scrums'' (SoS)}. The SoS is a ``team of
-teams,''\footnote{McChrystal, General Stanley and Collins, Tantum and
-Silverman, David and Fussell, Chris, Team of teams: New rules of engagement
-for a complex world, Penguin, 2015} which hold a \textbf{Scaled Daily Scrum
-(SDS)} event with a representative from each team (usually the team's Scrum
-Master, although any person or number of people may attend). The SDS exists
-to coordinate teams and remove impediments to delivering value.
-
-The SDS event mirrors the Daily Scrum in that it optimizes the
-collaboration and performance of the network of teams. Additionally, the
-SDS:
+A set of the teams that have a need to coordinate comprise a \textbf{``Scrum of Scrums'' (SoS)}. This team is a Scrum Team unto itself which is responsible for a fully integrated set of potentially shippable increments of product at the end of every Sprint from all participating teams. A SoS functions as a Release Team and must be able to directly deliver value to customers. To do so effectively, it needs to be consistent with the Scrum Guide; that is, have its own roles, artifacts, and events:
+
+Roles:
+
+It needs to have all of the skills necessary to deliver a fully integrated potentially shippable product at the end of every Sprint. (It may need experienced architects, QA Leaders, and other operational skill sets.) Since the SoS needs to be responsive in real-time to impediments raised by participating teams, the Scrum Masters of the participating teams need to be on the Scrum of Scrums.
+It has Product Owner representation to resolve prioritization issues.
+The Scrum Master of the Scrum of Scrums is called the Scrum of Scrums Scrum Master (SoSM).
+
+Events:
+
+A Backlog Refinement event wherein they decide what impediments are “ready” to be removed, how best to remove them, and how the team will know it is “done.”
+Particular attention should be paid to the SoS Retrospective in which the teams’ representatives share any learnings or process improvements that their individual teams have succeeded with, in order to standardize those practices across the teams within the SoS.  
+The SoS holds a \textbf{Scaled Daily Scrum (SDS)}. The SDS event mirrors the Daily Scrum in that it optimizes the collaboration and performance of the network of teams. Beside the Scrum Masters, any person or number of people from participating teams may attend.
+
+Additionally, the SDS:
 
 \begin{itemize}
 \item is time-boxed to 15 minutes or less.
-\item must be attended by a representative of each team.
-\item is a forum where team representatives address 3 simple questions:
+\item must be attended by a representative of each team including the Product Owner team and relevant Communities of Practice.
```

Comment:
Communities of Practice are not a required element of Scrum@Scale and are not a component in the framework.  We don't think they should be listed as a required attendee of the SDS.  Nothing in the guide has told you what a community of practice is.  We could replace with "relevant subject matter experts" or we could just delete.  These people don't need to come every time?